package.json

```diff
@@ -45,14 +55,15 @@
     "@types/jsdom": "2.0.*",
     "@types/node": "~9.6.5",
     "@types/sinon": "^1.16.31",
+    "codecov": "3.5.0",
+    "cpx": "1.5.0",
+    "cross-spawn": "6.0.5",
     "execa": "^0.9.0",
-    "grunt": "^1.0.1",
-    "grunt-dojo2": "~4.0.0",
-    "grunt-tslint": "^5.0.1",
+    "glob": "7.1.4",
     "husky": "2.2.0",
     "jsdom": "^9.5.0",
     "lint-staged": "8.1.6",
-    "load-grunt-tasks": "^3.5.2",
+    "npm-run-all": "^4.1.5",
```

Comment:
Is there any reason this shouldn't be pinned?

Reply:
missed it! thanks!